File: store_agent/src/main/java/com.abc/dao/ScoreDAO.java
package com.abc.dao;

import com.abc.model.VendorResponse;
import com.abc.model.rest.DecisionResponse;
import com.abc.model.rest.UserRequest;

public interface ScoreDAO {

    void saveUserRequest(UserRequest request);

    void storeVendorResponse(VendorResponse[] vendorResponse);

    void storeDecision(DecisionResponse decision);

}

File: src/main/java/com.abc/dao/ScoreDAO.kt
package com.abc.dao

import com.abc.model.VendorResponse
import com.abc.model.rest.DecisionResponse
import com.abc.model.rest.UserRequest

interface ScoreDAO {

    fun saveUserRequest(request: UserRequest)

    fun storeVendorResponse(vendorResponse: VendorResponse)

    fun storeDecision(decision: DecisionResponse)

}